Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -51,5 +70,23 @@ def symlink_ld_so
       FileUtils.ln_sf ld_so, brew_ld_so
     end
     private_class_method :symlink_ld_so
+
+    def symlink_gcc_libs
+      gcc_optprefix = HOMEBREW_PREFIX/"opt"/OS::LINUX_PREFERRED_GCC_FORMULA
+
+      GCC_RUNTIME_LIBS.each do |library|
+        gcc_library = gcc_optprefix/"lib/gcc"/GCC_VERSION_SUFFIX/library
+        gcc_library_symlink = HOMEBREW_PREFIX/"lib"/library
+        # Skip if the link target doesn't exist.
+        next unless gcc_library.readable?
+
+        # Also skip if the symlink already exists.
+        next if gcc_library_symlink.readable?
+
+        FileUtils.mkdir_p HOMEBREW_PREFIX
```

Comment:
Won't this directory already exist if `gcc_library.readable?` is true?

Reply:
Yes, I'm actually not sure there's any circumstance where this wouldn't exist before reaching this point. I'm going to remove this line.

Reply:
Or raise an exception if it doesn't exist.